tests/rules/javascript/test.content_scripts_file_absent.js

```diff
@@ -35,36 +35,30 @@ describe('content_scripts_file_absent', () => {
 
   it('should not report any errors when filename is not static string', async () => {
     const code = `
-      const fileName = 'script.js';
-      browser.tabs.executeScript({file: \`/content_scripts/\${fileName}\`});
-      browser.tabs.executeScript({file: '/content_scripts/'+ fileName });
-      browser.tabs.executeScript({file: '/content_scripts/' + 'absentFile.js' });
-      browser.tabs.executeScript(1,{file: '/content_scripts/' + 'absentFile.js' });
-      browser.tabs.executeScript(1,{file: 2 });
-      browser.tabs.executeScript(1,{file: null });
+      browser.tabs.executeScript({ file: \`/content_scripts/\${fileName}\` });
```

Comment:
Nit, I would put some inline comments inside this template string to make it more clear which are the grouped the scenarios, e.g.:

```
// API calls using a non literal strings file attribute.
....

// API calls with a non string file attribute.
...

// API calls without a file attribute.
...

// non WebExtensions API function calls.
...
```